apps/public-docsite/.eslintrc.json

```diff
@@ -4,6 +4,7 @@
   "rules": {
     "@typescript-eslint/no-explicit-any": "off",
     "deprecation/deprecation": "off",
+    "import/no-extraneous-dependencies": ["error", { "devDependencies": true, "packageDir": [".", "../.."] }],
```

Comment:
this doesn't work for our use case, thus should not be used like this. 
- this now forces to turn off this rule on some imports which leads to confusing DX

we need to enable plugin for transitive dependencies. https://github.com/microsoft/fluentui/issues/21999